src/components/screens/vacation-requests-screen.tsx

```diff
@@ -73,6 +81,58 @@ const VacationRequestsScreen = () => {
     fetchVacationRequestStatuses();
   }, [vacationRequests]);
 
+    /**
+   * Initialize logged in person data.
+   */
+  const getPersonVacations = async () => {
+    if (persons.length) {
+      setLoading(true);
+      if (loggedInPerson || config.person.id) setLoading(false);
+      {
+        const fetchedPersons = await personsApi.listPersons({ active: true });
+        setPersons(fetchedPersons);
+      }
+    }
+  };
+
+  useMemo(() => {
+    if (!persons) {
+      getPersonVacations();
+    }
+  }, [persons]);
```

Comment:
Apply the same suggestions as in the vacations-card.tsx

Reply:
Added.